deployability/modules/testing/tests/test_central_components/test_stop.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2015, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import pytest
+
+from ..helpers.generic import HostInformation, GeneralComponentActions
+from ..helpers.manager import WazuhManager
+from ..helpers.dashboard import WazuhDashboard
+from ..helpers.indexer import WazuhIndexer
+from modules.testing.utils import logger
+
+
+@pytest.fixture(scope="module", autouse=True)
+def wazuh_params(request):
+    wazuh_version = request.config.getoption('--wazuh_version')
+    wazuh_revision = request.config.getoption('--wazuh_revision')
+    dependencies = request.config.getoption('--dependencies')
+    targets = request.config.getoption('--targets')
+
+    params = {
+        'wazuh_version': wazuh_version,
+        'wazuh_revision': wazuh_revision,
+        'dependencies': dependencies,
+        'targets': targets
+    }
+    yield params
+    logger.info('Restoring Central Component status')
```

Comment:
the Wazuh central components

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/5289/commits/7c1d2270fe036e068551866cd700baa9e1b5a2ea